_episodes/02-motivation.md

```diff
@@ -184,27 +164,26 @@ This applies directly to the example of working on article revisions - will you
 For instance, if a colleague cannot find what data goes with which figures, there are high chances that you will also be unable to find it three years from now.
 In addition, itt is not uncommon to modify the design of the figures multiple times (sometimes back and forth), often modifying all figures at once.
 
->Once a reviewer ask me to overlay individual data points onto all our 5 boxplots figures.
+> Once a reviewer ask me to overlay individual data points onto all our 5 boxplots figures.
 The project was an old one, and I had not touched the data for years.
 Finding the right data and redo the all 5 figures would usually take ages using SPSS or excel.
-But since I used code, I had all figures 15 minutes later.
+> But since I used code, I had all figures 15 minutes later.
 (Note, after seeing the new figures, the reviewer agreed that the original version was better).
->{: .testimonial}
+> {: .testimonial}
 
 Later on in the project, community advantages are coming in.
 Data and code reusability is not only a mark of research transparency and robustness, it also means you can reuse your own code and data.
 It also means you can reuse code and data produced by other researchers.
-The snow ball effect may be huge, and the objective of this course is to allow you to do **better science in less time** ( https://www.nature.com/articles/s41559-017-0160:)
-
-
-
->As an example it was estimated that research data management  takes about 5% of your time, on the other hand, time lost due to poor data management is estimated to be 15%. 
-
+The snow ball effect may be huge, and the objective of this lesson is to allow you to do **better science in less time** ( https://www.nature.com/articles/s41559-017-0160:)
 
+> As an example it was estimated that research data management  takes about 5% of your time, on the other hand, time lost due to poor data management is estimated to be 15%. 
 
 
 ### Team and community building
 
+<img src="../fig/research-foundation.jpg" alt="A house representing machine learing and AI is set upon bricks that one person is sliding below the house. On the bricks, we can read data science principles like open science, backups, reproducibiliy, and FAIR principles." width="500"/>
+
+_Data science foundations. The Turing Way project illustration by Scriberia for The Turing Way Community Shared under CC-BY 4.0 License. Zenodo. http://doi.org/10.5281/zenodo.3332807_
```

Comment:
wondering if this is the right place for this figure, (it is also in the introduction) ?

Reply:
Moved the images around. I can't see the preview but let me see how I an share.

Reply:
I can serve it on my computer without problems

Reply:
Great! do feel free to move around images. There are more to add.